configurations/compression/lz4_dict_compression.yaml

```diff
@@ -0,0 +1,6 @@
+post_prepare_cql_cmds: >-
```

Comment:
still I'd add some comment about speculative_retry just in case someone wants to use it in typical test and knowing why this is here.